hms-app/src/pages/IdeationPortal.tsx

```diff
@@ -30,7 +30,7 @@ function IdeationPortal() {
   const loadHackathons = () => {
     getListOfHackathons(instance).then((data) => {
       const upcomingHackathon = data.find(
-        (hackathon) => hackathon.startDate < VALID_DATE
+        (hackathon) => hackathon.startDate < MIN_DATE
```

Comment:
if the start date is BEFORE the MIN_DATE then it would be a very old (and likely invalid) hackathon

Reply:
thats to find the one hackathon with "null date"